internal/kubeclient/kubeclient_test.go

```diff
@@ -974,21 +974,6 @@ func TestUnwrap(t *testing.T) {
 		testUnwrap(t, execClient, serverSubjects)
 	})
 
-	t.Run("gcp client", func(t *testing.T) {
```

Comment:
Why were these test blocks removed (`gcp client` and `azure client`)?

Reply:
These don't exist anymore in the k8s libraries starting in 1.26. This vendor-specific auth code was removed, so they don't compile anymore.

Reply:
See https://github.com/kubernetes/kubernetes/blob/60d73ba75148483d150622a4d8b6e122e189ceab/staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure_stub.go#L33-L35 and https://github.com/kubernetes/kubernetes/blob/60d73ba75148483d150622a4d8b6e122e189ceab/staging/src/k8s.io/client-go/plugin/pkg/client/auth/gcp/gcp_stub.go#L33-L35

Reply:
Sounds good. We still test the `regular client`, `oidc client`, and the `exec client`. 

`oidc client` and `exec client` are now the only supported plugins in 1.26+: https://github.com/kubernetes/kubernetes/tree/v1.26.0/staging/src/k8s.io/client-go/plugin/pkg/client/auth